apps/frontend/src/utils/sendRequest.ts

```diff
@@ -9,6 +10,19 @@ import {
 import { FormData } from '../types/formData';
 import { AdminNotes, formSchema } from '../types/formSchema';
 
+const authenticatedAxios
+= axios.create();
```

Comment:
nit: fix the whitespace / unnecessary line break here

Reply:
Fixed